tasks/Functions-admin/setNodeKey.js

```diff
@@ -0,0 +1,37 @@
+const { VERIFICATION_BLOCK_CONFIRMATIONS, networkConfig } = require("../../network-config")
+
+task("functions-set-node-key", "Sets the per-node public key in the Functions oracle contract")
+  .addParam("key", "Node-assigned public key")
+  .addOptionalParam("node", "Address of the node for which the public key is to be set (defaults to msg.sender)")
+  .setAction(async (taskArgs) => {
+    if (network.name === "hardhat") {
+      throw Error("This command cannot be used on a local development chain.  Specify a valid network.")
+    }
+
+    let overrides = undefined
+    if (network.name === "goerli") {
+      overrides = {
+        // be careful, this may drain your balance quickly
+        maxPriorityFeePerGas: ethers.utils.parseUnits("50", "gwei"),
+        maxFeePerGas: ethers.utils.parseUnits("50", "gwei"),
```

Comment:
I would wish this param to be configured globally. Please create a ticket for this.